src/main/java/org/openstreetmap/atlas/tags/annotations/extraction/AltitudeExtractor.java

```diff
@@ -0,0 +1,41 @@
+package org.openstreetmap.atlas.tags.annotations.extraction;
+
+import java.util.Optional;
+
+import org.openstreetmap.atlas.geography.Altitude;
+import org.openstreetmap.atlas.tags.annotations.Tag;
+import org.openstreetmap.atlas.utilities.scalars.Distance;
+
+/**
+ * Extracts a {@link Altitude} from a value. Can be used on tag values such as
+ * {@link org.openstreetmap.atlas.tags.HeightTag}.
+ *
+ * @author bbreithaupt
+ */
+public class AltitudeExtractor implements TagExtractor
+{
+    /**
+     * Validates and converts a value to a {@link Distance}.
```

Comment:
Shouldn't this be `Validates and converts a value to a {@link **Altitude**}?

Reply:
You are correct. That was a copy paste error.